provisioning_client/tests/hsm_client_riot_ut/hsm_client_riot_ut.c

```diff
@@ -107,15 +112,18 @@ static char* umocktypes_stringify_RIOT_ECC_PRIVATE(const RIOT_ECC_PRIVATE* value
     }
     else
     {
-        int length = snprintf(NULL, 0, "{ %p }", value->data);
+        int length = snprintf(NULL, 0, "{ %d, %zd, %p }",
+            value->s, value->n, value->p);
         if (length < 0)
         {
             result = NULL;
         }
         else
         {
-            result = (char*)my_gballoc_malloc(length + 1);
-            (void)snprintf(result, length + 1, "{ %p }", value->data);
+            length++;
+            result = (char*)my_gballoc_malloc(length);
```

Comment:
Check the `result` to make sure it isn't NULL as above line 107-108 does

Reply:
Fixed